### DISCUSSION TITLE
Split the storage of metric and trace ?

### DISCUSSION THREAD

**Discussion by shichaoyuan** [apache/skywalking#10319] [General]
In skywalking there are two types of data, metrics and trace, which are now stored in one storage backend.
The two have different characteristics, is split storage more appropriate?

**Comment by wu-sheng** [comment]
This was discussed repeatedly. There is no point to increase the complexity to get few benefits.
BanyanDB would be native first class database choice in the end of 2023.

**Reply by shichaoyuan** [reply]
ok
Looking forward to BanyanDB
